oshi-core/src/main/java/oshi/software/os/unix/freebsd/FreeBsdOSProcess.java

```diff
@@ -339,10 +344,7 @@ private boolean updateAttributes(String[] split) {
         this.name = this.path.substring(this.path.lastIndexOf('/') + 1);
         this.minorFaults = ParseUtil.parseLongOrDefault(split[15], 0L);
         this.majorFaults = ParseUtil.parseLongOrDefault(split[16], 0L);
-        long nonVoluntaryContextSwitches = ParseUtil.parseLongOrDefault(split[17], 0L);
-        long voluntaryContextSwitches = ParseUtil.parseLongOrDefault(split[18], 0L);
-        this.contextSwitches = voluntaryContextSwitches + nonVoluntaryContextSwitches;
-        this.commandLine = split[19];
+        this.commandLine = split[17];
```

Comment:
I've never really liked the numeric offsets in the split.  I think there's room to really improve this by making the `PS_COMMAND` list in this PR an ENUM.  That gives you:
 - built in immutability so you don't have to turn the list unmodifiable
 - strong correspondence between the enum ordinal and array lookup value here, robust to reordering

Reply:
I think this is a great idea for a future refactoring.